packages/fast-development-site-react/src/components/site/component-wrapper.tsx

```diff
@@ -41,7 +41,7 @@ const styles: ComponentStyles<IComponentWrapperManagedClasses, IDevSiteDesignSys
         borderBottom: componentWrapperBorder,
     },
     componentWrapper__transparent: {
-        background: `transparent ${toPx(8)}/${toPx(8)} ${checker} repeat`
+        background: `${checker}`
```

Comment:
You don't need template literals here since there are no calculations inside to be made, it can just be `background: checker`.